src/octopeer-github/main/Settings/DoNotWatchSettings.ts

```diff
@@ -0,0 +1,88 @@
+/// <reference path="../../content/ElementSelectionBehaviour/NameElementSelectionBehaviour/OtherContributerNameElementSelectionBehaviour.ts"/>
+/// <reference path="../../content/ElementSelectionBehaviour/NameElementSelectionBehaviour/PRCreatorNameElementSelectionBehaviour.ts"/>
+/// <reference path="../../content/ElementSelectionBehaviour/NameElementSelectionBehaviour/PRParticipantNameElementSelectionBehaviour.ts"/>
+/// <reference path="../../content/ElementSelectionBehaviour/SettingElementSelectionBehaviour/AssigneeSettingElementSelectionBehaviour.ts"/>
+/// <reference path="../../content/ElementSelectionBehaviour/SettingElementSelectionBehaviour/LabelSettingElementSelectionBehaviour.ts"/>
+/// <reference path="../../content/ElementSelectionBehaviour/SettingElementSelectionBehaviour/LockConversationSettingElementSelectionBehaviour.ts"/>
+/// <reference path="../../content/ElementSelectionBehaviour/SettingElementSelectionBehaviour/MilestoneSettingElementSelectionBehaviour.ts"/>
+/// <reference path="../../content/ElementSelectionBehaviour/SettingElementSelectionBehaviour/UnsubscribeSettingElementSelectionBehaviour.ts"/>
+/// <reference path="../../content/ElementSelectionBehaviour/TabHeaderElementSelectionBehaviour/CommitsTabHeaderElementSelectionBehaviour.ts"/>
+/// <reference path="../../content/ElementSelectionBehaviour/TabHeaderElementSelectionBehaviour/ConversationTabHeaderElementSelectionBehaviour.ts"/>
+/// <reference path="../../content/ElementSelectionBehaviour/TabHeaderElementSelectionBehaviour/FilesChangedTabHeaderElementSelectionBehaviour.ts"/>
+/* tslint:enable:max-line-length */
+
+/**
+ * Created by Youri on 19-5-2016.
+ * Additional type for storing tuples of Element and Event Creatables.
+ */
+type ElementXEventCreatable = {
+    element: ElementSelectionBehaviourCreatable,
+    event: ElementEventBindingCreatable
+};
+
+/**
+ * Class for indicating all internal settings of the application.
+ */
+const DoNotWatchSettings = new (class DoNotWatchSettings {
+    /**
+     * Gets Elements not to Log, from the chrome storage.
+     * @returns {ElementSelectionBehaviourCreatable[]}
+     */
+    public getElements() {
+        let doNotWatchElements: ElementSelectionBehaviourCreatable[];
+
+        return doNotWatchElements;
+    }
+    /**
+     * Gets Events not to Log, from the chrome storage.
+     * @returns {ElementSelectionBehaviourCreatable[]}
+     */
+    public getEvents() {
+        let doNotWatchEvents: ElementEventBindingCreatable[];
+        doNotWatchEvents = [];
+
+        if (Settings.getFocus()) {
+            doNotWatchEvents.push(ScrollIntoViewElementEventBinding);
+            doNotWatchEvents.push(ScrollOutOfViewElementEventBinding);
+        }
+
+        return doNotWatchEvents;
+    }
+    /**
+     * Gets Elements not to Log, from the chrome storage.
+     * @returns {ElementSelectionBehaviourCreatable[]}
+     */
+    public getCombinations() {
+        let doNotWatchCombination: ElementXEventCreatable[];
```

Comment:
Isn't this local variable 'doNotWatchCombination' redundant?
Or will you modify this later on?